testkit-backend/lib/testkit/backend/messages/requests/result_consume.rb

```diff
@@ -4,14 +4,14 @@ class ResultConsume < Request
       def process
         named_entity('Summary',
                      **{
-                       # serverInfo: to_map(summary.server, :protocol_version, :address, :agent),
-                       serverInfo: to_map(summary.server, :protocol_version, :agent),
+                       serverInfo: to_map(summary.server, :protocol_version, :address, :agent),
+                       # serverInfo: to_map(summary.server, :protocol_version, :agent),
                        counters: to_map(summary.counters, *%w[constraints_added constraints_removed contains_system_updates contains_updates indexes_added
           indexes_removed labels_added labels_removed nodes_created nodes_deleted properties_set relationships_created
           relationships_deleted system_updates]),
                        query: { text: summary.query.text, parameters: summary.query.parameters.transform_values(&method(:to_testkit)) },
                        database: summary.database.name,
-                       queryType: 'rw',
+                       queryType: Neo4j::Driver::Ext::Internal::Summary::InternalResultSummary.query_type(summary.query_type),
```

Comment:
testkit-backend should not have any references to implementation specifics. `Ext` is jruby specific and cannot be used here. We are overriding the `query_type` method to use it here in a simple way.

Reply:
```suggestion
                       queryType: summary.query_type,
```